web/src/components/tasks/LicenseTask.test.tsx

```diff
@@ -215,11 +218,11 @@ describe("<LicenseTask />", () => {
 
     it("does not display error alert when license status is found", async () => {
       renderTask();
-
       mockApi.checkLicenseStatus.mockResolvedValue(generateUserData({}));
```

Comment:
For this test, nothing has changed but the expected results changed. I initial thought is that it shouldn't change from pending to completed